composer.json

```diff
@@ -1,10 +1,32 @@
 {
+	"name": "inblockio/data-accounting",
```

Comment:
Shouldn't this be `inblockio/DataAccounting`?

Reply:
mediawiki expects all lowercase and a dash as separator for words in the repo name:
https://www.mediawiki.org/wiki/MediaWiki_Packagist

Reply:
Should the GH repo name be renamed to `inblockio/data-accounting` as well?

Reply:
that is fine as it is. MediaWiki would expect an extension to be named like this "DataAccounting"